js/src/forum/components/LogInButtons.js

```diff
@@ -20,6 +21,28 @@ export default class LogInButtons extends Component {
    * @public
    */
   items() {
-    return new ItemList();
+    const items = new ItemList();
+
+    const providers = app.forum.data.attributes.ssoProviders;
+
+    for (const provider in providers) {
+      const className = `Button LogInButton--${provider}`;
+      const path = `/auth/${provider}`;
+      var style = "";
```

Comment:
Please use an object for styles. (`style = {}`, `style.beckgroundColor = ''`, ...). This makes it easier for other exensions to override parts of the styles. Also it might be more performant for Mithril because it doesn't need to override the whole style, it can only update the properties that changed.

And replace that `var` with `let` :stuck_out_tongue: (or `const` if you replace with an object).

Reply:
Done, didn't know that was an option, thanks!